### PR TITLE
Makes the Documentation tag give 1 GBP

### PR DESCRIPTION
**Pull request by SuperNovaa41**
## Why It's Good For The Coders
Documentation is extremely important for the longevity of the codebase.
Well written documentation allows for experienced coders to use the code to it's fullest extent, while also allowing new contributors to get a foothold easier.
Code improvement gives 2GBP, and you can get that for changing `var/mob/m` to `var/mob/mob`. Documentation tends to require a little more work. As such, I feel that people should be rewarded for taking their time to document someone else's code.

### CODE REVIEW

**Comment by capsaicinz** [comment]
real?

**Comment by MacBlaze1** [comment]
REAL

**Comment by BallastMonster** [comment]
real?

File: .github/gbp.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ reset_label = "GBP: Reset"
 "Atomic" = 2
 "Balance/Rebalance" = -5
 "Code Improvement" = 2
+"Documentation" = 2
```
Comment (Kylerace) [inline .github/gbp.toml:10]: i think documentation should be less than code improvement

Comment (SuperNovaa41) [reply .github/gbp.toml:10]: thats okay with me

**Comment by Kylerace** [comment]
@tgstation/commit-access

**Comment by Mothblocks** [comment]
I don't understand what giving this 1 GBP will accomplish.

We always give these code improvement anyway. Code improvement + Documentation is *two* GBP, not three. High vs. low.

**Comment by SuperNovaa41** [comment]
@Mothblocks if you look at the merged prs with the documentation label over half of them don't have code improvement, just documentation

**Comment by Mothblocks** [comment]
That's depressing

**Comment by optimumtact** [comment]
Either im mistaken or this used to give GBP, I specifically asked that it did in the original implementation.

**Comment by MrMelbert** [comment]
I remember documentation giving GBP too, tbh I didn't even realize it didn't right now

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
> Either im mistaken or this used to give GBP, I specifically asked that it did in the original implementation.

@optimumtact is right here.

**Pull request merged**